inscriptions/ISic020371.xml

```diff
@@ -93,7 +93,7 @@
                                 <!-- Based on Glyph table: (a) alpha04; gamma02(?); reverse gamma02 (?); epsilon23; heta05; iota01; omicron04; mu02; nu05; rho17; sigma03; tau07; (b) alpha04; heta05; epsilon25; epsilon17; nu05; omicron04; rho19; sigma03; upsilon03.-->
                                 <locus from="line1" to="line1">Line 1</locus>
                                 <dimensions type="letterHeight">
-                                    <height unit="mm"/>
+                                    <height unit="mm">face a, 5-8; face b, 4-7</height>
```

Comment:
put face a as one locus and face b another locus, separately? (you don't have to include the from and to line element, and if there are (in due course) xml:id on the face text sections, locus could take @target instead with @target="#facea" or similar. For now, you can just do `<locus>Face a</locus>`